flocker/node/script.py

```diff
@@ -600,7 +616,11 @@ def get_api(self):
         if backend.needs_reactor:
             api_args = api_args.set("reactor", self.reactor)
 
-        return backend.api_factory(**api_args)
+        try:
+            return backend.api_factory(**api_args)
+        except StorageInitializationError as e:
+            if e.code == StorageInitializationError.CONFIGURATION_ERROR:
+                raise UsageError(u"Configuration error", *e.args)
```

Comment:
Shouldn't this re-raise if it has a different code? If not, please add a comment explaining why it's OK to swallow the error.